src/vaults/vaultModels.ts

```diff
@@ -2,12 +2,12 @@ import { DistributionStats } from "../common/commonModels";
 
 export interface CollateralTimeData {
     date: number;
-    amount: number;
+    amount: string;
```

Comment:
Do you want to have the amount as a string here rather than a a `Big` if it's denoted in DOT or `BN` if it's denoted in planck?

Reply:
Dangit I thought I went through to check any other changes where I used a string, but missed this file. Will do.